flask_login.py

```diff
@@ -863,7 +863,9 @@ def _cookie_digest(payload, key=None):
 def _get_remote_addr():
     address = request.headers.get('X-Forwarded-For', request.remote_addr)
```

Comment:
Why not use the [access_route](http://werkzeug.pocoo.org/docs/0.10/wrappers/#werkzeug.wrappers.BaseRequest.access_route) attribute off the `Request` object? That is a list and already includes `request.remote_addr` and the `X-Forwarded-For` addresses.